packages/router/src/apply_redirects.ts

```diff
@@ -161,7 +161,7 @@ class ApplyRedirects {
             throw e;
           }));
         }),
-        concatAll(), first((s: any) => !!s), catchError((e: any, _: any) => {
+        first((s: any) => !!s), catchError((e: any, _: any) => {
```

Comment:
Since you are touching, this saves you a closure and some bytes:
```suggestion
        first(Boolean), catchError((e: any, _: any) => {
```

Reply:
The code at this location is already different on `master` so I'm just going to leave it as-is here to limit the diff. Thanks for the suggestion though!